apps/meteor/app/ui-message/client/messageBox/messageBox.html

```diff
@@ -24,7 +24,7 @@
 							{{> icon block="rc-input__icon-svg" icon="emoji"}}
 						</span>
 					{{else}}
-						<div class="rc-message-box__icon"></div>
+						<div class="rc-message-box__federation_icon"></div>
```

Comment:
Shouldn't the class be `rc-message-box__icon rc-message-box__federation_icon`?

Reply:
I've created this new class because the old one was showing the cursor as a pointer, which was leading to weird behavior ( cursor as pointer where there is nothing (emoji picker)).